lab-anthony/lib/storage.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
```

Comment:
This is super duper nit-picky, and very stylistic, so you should feel free to disagree. A convention that some people follow, including the angular community, would be to attach all your functions to exports in one place, then write the function definitions below, that way it's clear up front what's being attached if you just want to give it a cursory glance without the function body breaking it up. Basically, something like this: 
```javascript 
exports.setItem = setItem;
exports.fetchItem = fetchItem;
...

function setItem(){
// blah blah 
}

function fetchItem(){
// blah blah 
}
...
```